pkg/config/config.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	yaml "gopkg.in/yaml.v2"
 
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
```

Comment:
Seems like this import creates a cycle:
```
package github.com/DataDog/datadog-agent/cmd/agent
	imports github.com/DataDog/datadog-agent/cmd/agent/app
	imports github.com/DataDog/datadog-agent/cmd/agent/api
	imports github.com/DataDog/datadog-agent/cmd/agent/api/agent
	imports github.com/DataDog/datadog-agent/cmd/agent/api/response
	imports github.com/DataDog/datadog-agent/pkg/autodiscovery/integration
	imports github.com/DataDog/datadog-agent/pkg/util/containers
	imports github.com/DataDog/datadog-agent/pkg/config
	imports github.com/DataDog/datadog-agent/cmd/agent/common
	imports github.com/DataDog/datadog-agent/pkg/autodiscovery
	imports github.com/DataDog/datadog-agent/pkg/autodiscovery/configresolver
	imports github.com/DataDog/datadog-agent/pkg/autodiscovery/listeners
	imports github.com/DataDog/datadog-agent/pkg/config
```

Not sure how to fix this though 😞.

Reply:
yes I will, define the defaultSecurityAgentLogsFile in another package, since it is only used in config now

Reply:
@KSerrania, I finally remove the circular dependency issue.